sonarqube/README.md

```diff
@@ -13,6 +13,50 @@ No additional installation is needed on your server.
 
 ### Configuration
 
+SonarQube exposes metrics from two sources: its web API and JMX. To collect all of the
+[metrics specified below](#metrics), you'll need to configure two intances of this check. One instance will monitor
+SonarQube's JMX beans and the other will monitor SonarQube's Web API.
+
+Documentation on SonarQube's Web API is available at `/web_api` on your SonarQube web UI. By default this integration
+collects all relevant SonarQube performance metrics exposed through SonarQube's JMX beans. The configuration for these
+default metrics is available in the [sonarqube.d/metrics.yaml][12] file. Documentation on these beans is available on
+[SonarQube's website][13].
+
+SonarQube's JMX server is not enabled by default. More information on how to enable and configure JMX within SonarQube
+in available within the [SonarQube documentation][14].
+
+This is a basic `sonarqube.d/conf.yaml` example based on SonarQube and JMX defaults and can be used a starting point
+when configuring for both the host-based or container-based agent installation.
+
+```yaml
+init_config:
+    is_jmx: false
+    collect_default_metrics: true
+instances:
+  # Web API instance
+  - is_jmx: false
+    web_endpoint: http://localhost:9000
+    auth_type: basic
+    username: <username> # Defined in the Web UI
+    password: <password> # Defined in the Web UI
+    components:
+      my-project:
+        tag: my-branch
+  # JMX instance
+  - host: localhost
+    is_jmx: true
+    port: 10443
+    user: <username>      # Defined in SonarQube's sonar.properties file
+    password: <password>  # Defined in SonarQube's sonar.properties file
+```
+
+> Note: Once configured, you need to have Sonar scan at least one project for many of the metrics to populate in
+> Datadog.
+
+Projects in SonarQube often contain multiple SCM branches and tags. To specify what branch or tag for the integration
+to collect metrics from the Web UI, specify the `tag` configuration for the component, otherwise the default branch
+will be used (typically `master`).
```

Comment:
```suggestion
will be used (typically `main`).
```

Reply:
I know I'm assuming git here, but I've never heard of a `main` branch being used. Curious if this is specific to Sonar and I'm just lacking insight here

Reply:
Github is enabling folks to rename their `master` branch to `main` (or whatever) in an effort to use more inclusive terminology. I'm just putting the wish out into the world that `master` gradually disappears, and `main` is normalized.

Reply:
Love it! I'll update it to read `main`.